packages/kit/src/runtime/server/page/render.js

```diff
@@ -188,9 +190,9 @@ export async function render_response({
 		throw error;
 	}
 
-	if (validation_errors) {
+	if (form_value) {
 		try {
-			serialized.errors = devalue(validation_errors);
+			serialized.form = devalue(form_value);
```

Comment:
I guess we need to enforce this being JSONable, not sure if that's already happening elsewhere